documentation/explainers/price-feed-identifiers.md

```diff
@@ -0,0 +1,77 @@
+# Price feed identifiers
```

Comment:
Do you think this file belongs in explainers or tutorials? I could see an argument for either because parts feel like mini tutorials, but it's definitely not a single, linear instruction set that shows you how to do one thing.

Reply:
I think explainers makes more sense for what this document is covering. The tutorials can walk the user through getting something specific set up or running, e.g., using your own price on TD.sol on testnet is a better fit for a tutorial.